meta-lmp-base/recipes-bsp/efitools/efitools_git.bb

```diff
@@ -14,10 +14,11 @@ SRC_URI += " \
     file://build-keys-for-lockdown-only.patch \
     file://allow-local-auths.patch \
     file://lockdown.conf \
-    file://unlock.patch \
-    file://unlock.conf \
 "
 
+# UnLock needs the user keys
```

Comment:
I would add a little bit more on the commit log as it looks like to me that the patch does more that only build the keys
And I'm not sure I understand what "build a key" is
Do we need to verify if the key files exists?

Reply:
the tool (unlock.efi) requires that the user provides the UEFI keys in order for the tool to be built - because the keys are later on embedded/compiled_in in the coff image.